test/integration/asymmetric-encrypt-decrypt.int.c

```diff
@@ -140,6 +141,17 @@ test_invoke (TSS2_SYS_CONTEXT *sapi_context)
     LOG_INFO("Decrypt successed.");
 
     LOG_INFO("Asymmetric Encryption and Decryption Test Passed!");
+
+    rc = Tss2_Sys_FlushContext(sapi_context, sym_handle);
+    if (rc != TSS2_RC_SUCCESS) {
+        LOG_ERROR("Tss2_Sys_FlushContext failed with 0x%"PRIx32, rc);
+        return 99;
```

Comment:
Could we add some comments about the 99 & 77 return codes?

Reply:
done